auth/src/main/java/org/aerogear/auth/ICredential.java

```diff
@@ -0,0 +1,4 @@
+package org.aerogear.auth;
+
+public interface ICredential {
```

Comment:
@danielpassos @secondsun do we have any existing Java naming conventions for the AG android projects that we should follow?